bundles/io/org.eclipse.smarthome.io.voice/src/main/java/org/eclipse/smarthome/io/voice/text/AbstractRuleBasedInterpreter.java

```diff
@@ -228,7 +295,7 @@ protected Rule itemRule(Object headExpression) {
     protected Rule itemRule(Object headExpression, Object tailExpression) {
         Expression tail = exp(tailExpression);
         Expression expression = tail == null ? seq(headExpression, name())
-                : seq(headExpression, name(tail.getFirsts()), tail);
+                : seq(headExpression, name(tail), tail);
```

Comment:
I had read this code before and never understood what the getFirsts() was for. This change to name() is nicer now!